app/models/image.rb

```diff
@@ -2,7 +2,7 @@ class Image < ActiveRecord::Base
   has_many :items
 
   has_attached_file :img,
-    :styles => { menu_list: "150x150>", item_list: "400x400>" },
+    :styles => { item_manage_list: "75x75>", menu_list: "150x150>", item_list: "400x400>" },
```

Comment:
Line is too long. [92/80]<br>Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.